sim/core/buffs.go

```diff
@@ -868,3 +866,72 @@ func ReplenishmentAura(character *Character, actionID ActionID) *Aura {
 		},
 	})
 }
+
+func DemonicPactAura(character *Character, spellPowerBonus float64) *Aura {
+
+	return character.GetOrRegisterAura(Aura{
+		Label:    "Demonic Pact",
+		Tag:      "SpellPowerBuff",
```

Comment:
you should add the tag as a const and use that in all the auras instead of a magic string